index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Form Survey</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <link rel="stylesheet" href="styles.css">
+
+</head>
+
+<body>
+    <div class="container form-bg">
+        <h1 class="heading">Microverse Survey Form</h1>
+
+        <div class="row">
+            <div class="col">
+                <input type="text" class="input form-control" placeholder="First name" aria-label="First name">
+                <input type="text" class="input form-control" placeholder="Last name" aria-label="Last name">
+                <input type="email" class="input form-control" placeholder="Email Address" aria-label="Email Address">
+                <input type="number" class="input form-control" placeholder="Age(optional)" aria-label="Age">
+                <select class=" input form-select" aria-label="Default select example">
+                    <option selected>What best describes your role</option>
+                    <option value="1">Student</option>
+                    <option value="2">Teacher</option>
+                    <option value="3">Software Engineer</option>
+                    <option value="3">Cloud Engineer</option>
+                </select>
+                <div class="input form-control">
```

Comment:
I think it would be good if you used html semantic tags like sections as child elements to a Parent div tag.